core/src/components/item/test/basic/item.e2e.ts

```diff
@@ -10,5 +10,37 @@ configs().forEach(({ title, screenshot, config }) => {
 
       await expect(page).toHaveScreenshot(screenshot(`item-diff`));
     });
+
+    /**
+     * This behavior needs to be tested for all modes & directions
+     * Safe padding should stay on the same side when the direction changes
+     */
+    test('should have safe area padding', async ({ page }) => {
+      await page.setContent(
+        `
+        <style>
+          :root {
+            --ion-safe-area-left: 40px;
+            --ion-safe-area-right: 10px;
+          }
+        </style>
+        <ion-list>
+          <ion-item>
+            <ion-label>Item with helper</ion-label>
+            <div slot="helper">Helper</div>
+          </ion-item>
+
+          <ion-item>
+            <ion-label> Single line text that should have ellipses when it doesn't all fit in the item</ion-label>
+          </ion-item>
+        </ion-list>
+      `,
+        config
+      );
+
+      const list = page.locator('ion-list');
+
+      await expect(list).toHaveScreenshot(screenshot('safe-area'));
```

Comment:
```suggestion
      await expect(list).toHaveScreenshot(screenshot('item-safe-area'));
```

The test name should start with the component name. You will need to delete the screenshots that were added and run the update action again or rename them.